static/js/top_left_corner.js

```diff
@@ -79,34 +56,27 @@ export function handle_narrow_activated(filter) {
         if (filter_name === "starred") {
             filter_li = $(".top_left_starred_messages");
             filter_li.addClass("active-filter");
+            pm_list.handle_narrow_deactivated();
```

Comment:
These calls also seem very likely to be working around some other bug in your code -- we should not be needing to call a "handle_narrow_deactivated" function inside another `handle_narrow_activated` handler's code path.

Reply:
Hmm, fixed it by removing these and handle the other bug in #21726.